packages/ruleset/test/schema-description.test.js

```diff
@@ -4,15 +4,13 @@ const { makeCopy, rootDocument, testRule, severityCodes } = require('./utils');
 const rule = schemaDescription;
```

Comment:
The main changes in this file are to remove the property-related tests (moved to the new testcase) and to simplify the way we verify that the correct path is reported in various testcases.